justfile

```diff
@@ -0,0 +1,27 @@
+_default:
+    just --list
+
+release_dir := "themes"
+whiskers_cmd := "whiskers"
+themekit_cmd := "npm run build"
```

Comment:
I see that there's an `npm run build` here, but don't see any `package.json` file with a `build` script defined?

Reply:
This was for building with the cider-themekit which required cloning their repo. Would it be better to include the cider-themekit build files in this repo?

Reply:
Could you link the repo so I can take a look at it? It may be worth it to submodule the repository if we're relying on build scripts from there.

Reply:
Here it is: https://github.com/ciderapp/cider-themekit

Reply:
Thanks, I think we should add it as a submodule as we're depending on their install script. 

We're starting to use [Renovate](https://docs.renovatebot.com/) across the organisation which should help keep this submodule up-to-date with scheduled and automatic PRs.

Reply:
Added `cider-themekit` as a submodule and modified the justfile to accommodate the change.
Hopefully, I did this right.
https://github.com/catppuccin/cider/pull/2/commits/7ff414cf976b4573f4a38c31c9b75890b10404ac
https://github.com/catppuccin/cider/pull/2/commits/e46c3521722660122cceb340ffd614c7b2c464b3

Reply:
Thanks for adding it, I _think_ it looks fine :D 
We can always fix afterwards if it ends up being weird, thanks!